### PR TITLE
Upgrade to 2.18

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: fastboot-server.js [hunk 1]
```diff
@@ -36,7 +39,7 @@ cluster(function() {
   if (assetPath) {
     app.get('/', fastboot);
     app.use(staticGzip(assetPath)),
-    app.use(express.static(assetPath));
+      app.use(express.static(assetPath));
```
Comment (kellyselden) [inline fastboot-server.js]: accident?

File: fastboot-server.js [hunk 2]
```diff
@@ -1,21 +1,23 @@
+/* eslint-env node */
```
Comment (kellyselden) [inline fastboot-server.js]: This should be done in the eslint override

File: .travis.yml [hunk 3]
```diff
@@ -2,19 +2,27 @@ language: node_js
...
-  - export PATH=$HOME/.yarn/bin:$PATH
-  - yarn global add bower phantomjs-prebuilt
-  - phantomjs --version
-install:
-  - yarn
+  - npm config set spin false
+
 script:
-  - yarn test
+  - npm run lint:js
```
Comment (kellyselden) [inline .travis.yml]: you lost yarn stuff here.

**Review by kellyselden — CHANGES_REQUESTED** [review]
left some notes

File: fastboot-server.js (hunk 1 above)
Comment (locks) [reply fastboot-server.js]: @kellyselden prettier re-formatted because the previous line is a comma 😱

**Review by kellyselden — APPROVED** [review]

**Pull request merged**